packages/copyable/src/Copyable.tsx

```diff
@@ -175,15 +176,14 @@ export default function Copyable({
     }
 
     // Forward darkMode prop for future versions of Button that support it
-    const buttonRestProps: {} = { darkMode };
+    // const buttonRestProps: {} = { darkMode };
```

Comment:
why?

Reply:
Right now Button doesn't have a darkMode prop so I'm getting an error saying that darkMode is not a valid HTML attribute

Reply:
hm, can we do something like this:
```tsx
const buttonRestProps = 'darkMode' in Button.propTypes ? { darkMode } : {};
```

Reply:
👍